app/lib/efile/nj/nj1040_calculator.rb

```diff
@@ -173,6 +187,32 @@ def calculate_line_9
         number_of_line_9_exemptions * 6_000
       end
 
+      def line_59_primary
+        primary_excess = get_personal_excess(@intake.primary.ssn, :box14_ui_wf_swf, EXCESS_UI_WF_SWF_MAX)
+        primary_excess += get_personal_excess(@intake.primary.ssn, :box14_ui_hc_wd, EXCESS_UI_WF_SWF_MAX)
+        primary_excess
+      end
+
+      def line_59_spouse
+        if @intake.filing_status_mfj?
+          spouse_excess = get_personal_excess(@intake.spouse.ssn, :box14_ui_wf_swf, EXCESS_UI_WF_SWF_MAX)
+          spouse_excess += get_personal_excess(@intake.spouse.ssn, :box14_ui_hc_wd, EXCESS_UI_WF_SWF_MAX)
+          return spouse_excess
+        end
+        0
+      end
+
+      def line_61_primary
+        get_personal_excess(@intake.primary.ssn, :box14_fli, EXCESS_FLI_MAX)
+      end
+
+      def line_61_spouse
+        if @intake.filing_status_mfj?
+          return get_personal_excess(@intake.spouse.ssn, :box14_fli, EXCESS_FLI_MAX)
+        end
+        0
```

Comment:
[dust, could be pebble] noting that `0` returned here is truthy.. but i would expect from 

`@nj2450_spouse.calculate if line_59_spouse || line_61_spouse`

above that we'd want falsy, right?

(question applies to `line_59_spouse` too)

Reply:
an add-on, it might be a good idea to assert the behavior in `nj1040_calculator_spec` so it doesn't require parsing the truthiness from code?